### ISSUE TITLE
EMBEDDEDMAP must be initiated by default to use it in UPSERT

### ISSUE THREAD

**Issue by freeart** [orientechnologies/orientdb#8701]
## Steps to reproduce
```sql
CREATE CLASS TForm
CREATE PROPERTY `TForm`.`answer` EMBEDDEDMAP STRING
ALTER PROPERTY `TForm`.`answer` mandatory true
ALTER PROPERTY `TForm`.`answer` notNull true
UPDATE TForm
SET
	answer.name = :answername
UPSERT 
WHERE id = :id
```
Returns 
```
com.orientechnologies.orient.core.exception.OValidationException: The field 'TForm.answer' is mandatory, but not found on record
```
I can't do like
```sql
UPDATE TForm
SET
	answer = {"name" :answername}
UPSERT 
WHERE id = :id
```
Because it rewrites all data in answer.

**Comment by luigidellaquila** [comment]
Hi @freeart 

Good point, probably we can leverage the schema here and make the first statement work. I'll check it ASAP and let you know

Thanks

Luigi

**Comment by luigidellaquila** [comment]
Hi @freeart 

I just pushed a fix, it will be released with V 3.0.13

Thanks

Luigi
